build.gradle

```diff
@@ -30,7 +24,8 @@ def jettyVersion = '9.3.8.v20160314'
 def jerseyVersion = '2.22.2'
 
 dependencies {
-    compile 'org.terasology.engine:engine:1.0.0'
+    //compile 'org.terasology.engine:engine:1.0.0'
+    compile project(':engine')
```

Comment:
Yes, it uses the local engine. I followed the [facade tutorial](https://github.com/MovingBlocks/Terasology/wiki/Codebase-Structure#facades) so that when you have the main repository (MovingBlocks/Terasology) as root and the nested repository at facades/Server, the engine in the root repository is used. The [PC facade is set up in the same way](https://github.com/MovingBlocks/Terasology/blob/develop/facades/PC/build.gradle#L41).

I'm not sure how packaging/deployment should be managed. Probably we should make a package with the engine and the facade as for the PC facade?